lib/src/components/card_rounded.dart

```diff
@@ -35,6 +49,9 @@ class CardRounded extends StatelessWidget {
     this.height,
     this.width,
     this.onClose,
+    this.closeButtonAlignment = Alignment.topRight,
+    this.customButtonIcon,
+    this.padding,
```

Comment:
Setting a default in the constructor makes it clearer to the user, and makes for a more readable `build` method.
```suggestion
    this.padding = const EdgeInsets.symmetric(
              vertical: 12,
              horizontal: 12,
            ),
```